pkg/operator/constants/constants.go

```diff
@@ -47,6 +47,10 @@ const (
 	// secrets the operator knows will be required for the next minor release of OpenShift.
 	MissingSecretsForUpgradeReason = "ManualModeMissingSecrets"
 
+	// MissingUpgradeableAnnotationReason is used the cluster is not upgradeable due to
+	// the CCO's config object missing the appropriate annotation.
```

Comment:
It is intended for Manual mode for sure. But it was easy enough to move the place where we check for the annotation early in the Upgradeable() path so that non-Manual users could move forward in the event that CCO's Upgradeable() code was giving false negatives.

Reply:
Ooo I like that, ok cool.